### DISCUSSION TITLE
Although I open "Minimise to notification area", there is nothing in the notification area, making my progress uncontrollable

### DISCUSSION THREAD

**Discussion by Arc-2023** [microsoft/terminal#17657] [Q&A] [answered]
What should I do to get my progress back after accidentally opening "Auto-Hide Window"?
It's still running in the background, but I can't see it in Terminal.
Now I can only see it through Task Manager:

BTW, i used it to run a python script, now the script is still running, and i don't want to restart it , because it has been running for a day already😥 
![image](https://github.com/user-attachments/assets/b2826afb-14b6-47d6-af6e-308324bc2569)
![image](https://github.com/user-attachments/assets/9174cbc8-f126-45a0-9bc4-7b3f33e8baff)

**Comment by DHowett** [comment]
Hmm. If you launch Terminal again, you can get into the settings and turn on the option to always show the icon in the notification area. It should make the icon appear, and you can right-click it and restore your other window.

**Reply by Arc-2023** [reply]
> Hmm. If you launch Terminal again, you can get into the settings and turn on the option to always show the icon in the notification area. It should make the icon appear, and you can right-click it and restore your other window.

thanks for answering, i open a new terminal instance but i got nothing in notification area
![image](https://github.com/user-attachments/assets/d56c98a4-64eb-4cdb-8705-f4faf76adee1)

**Comment by Arc-2023** [comment]
i also checked if the icon of terminal is open:
![image](https://github.com/user-attachments/assets/23773e77-1a9e-4dab-8abb-242789400836)

**Comment by Arc-2023** [comment]
![image](https://github.com/user-attachments/assets/d83b0def-643e-4d86-98ce-203d9379d3b7)

after clicking the "bring to front", it just opens a blank terminal

**Reply by DHowett** [reply]
If you open the Settings in the new terminal window, can you turn on “always display an icon in the notification area”? Then press Save.

**Reply by DHowett** [reply]
It is in the Appearance page.

**Reply by Arc-2023** [reply]
> If you open the Settings in the new terminal window, can you turn on “always display an icon in the notification area”? Then press Save.

Yes, I've opened it and toggled back and forth, there's still nothing in the notification area.

It's icon seemed to appear once in the past, but it just contains a blank terminal.

**Reply by DHowett** [reply]
Interesting. What if you try the command `wt -w 0 new-tab`. This is a command that will target the original window (assuming it is number 0).

**Reply by Arc-2023** [reply]
> Interesting. What if you try the command `wt -w 0 new-tab`. This is a command that will target the original window (assuming it is number 0).

Upon executing the aforementioned command, a new tab was opened.

**Reply by DHowett** [reply]
Ugh, sorry! What if you close the existing windows and use `wt -w -1` instead?

If that opens up the existing window, I’m not sure what else to do. Sorry ☹️ .

**Reply by Arc-2023** [reply]
> Ugh, sorry! What if you close the existing windows and use `wt -w -1` instead?
> 
> If that opens up the existing window, I’m not sure what else to do. Sorry ☹️

~~The command resulted in the opening of a new, empty terminal window. Furthermore, it became evident that the same problem was occurring in the issue area. If this is indeed a bug, it is hoped that a solution will be provided in the near future.😥~~

ohhhh, but `wt -w 1` actually worked!
it opens a new tab in the other window which contains the running process!

**Comment by Arc-2023** [comment, marked as answer]
Thanks to @DHowett for this suggestion.
In summary, running `wt -w 1` in a new tab can be an effective solution to the problem I encountered.
In other cases, other numbers may be effective instead of "1".

**Reply by DHowett** [reply]
YAY!
